resources/FAQ.md

```diff
@@ -6,7 +6,7 @@
 
 ### **2. Using a Private Node or Infura with the Fastlane Bot**
 - **Question:** Can a private node or Infura be used with the Fastlane Bot?
-- **Answer:** Yes. The bot can use any RPC. This can be edited in the fastlane_bot/config/providers file - search for: self.RPC_URL. Note that some functions require an Alchemy API key, making it necessary unless the functions themselves are modified. These functions are in fastlane_bot/helpers/txhelpers, and include get_access_list, submit_private_transaction, and get_max_priority_fee_per_gas_alchemy.
+- **Answer:** Yes. The bot can use any RPC. This can be edited in the fastlane_bot/config/providers file - search for: self.RPC_URL. Note that some functions require an Alchemy API key, making it necessary unless the functions themselves are modified. These functions are in fastlane_bot/helpers/txhelpers, and include _create_access_list, _send_private_transaction, and _max_priority_fee_per_gas.
```

Comment:
we should not refer to private functions in the documentation; by Python convention, functions starting with underscore should be considered private, meaning they are not part of the API and outsiders should not call them

Reply:
Initially I indeed just removed this part altogether, but then felt that it was nevertheless useful.
Neither one of these functions is used outside of that module, hence I have "privatized" them.
Removing the underscore would hide that fact, while keeping it makes this description flawed.
But this is a purely semantic issue, so resolving.